core/src/main/java/com/testaem/aem/core/models/impl/SinglePageNewsImpl.java

```diff
@@ -27,9 +36,60 @@ public class SinglePageNewsImpl implements NewsPage {
     @ValueMapValue(name = "image")
     private String imagePath;
 
+    @ValueMapValue
+    private String truncatedText;
+
     @ValueMapValue
     private Date date;
 
+    @Self
+    private SlingHttpServletRequest request;
+
+    public static String truncate1(String input, int size) {
```

Comment:
is this method or the method below used anywhere? also the naming is questionable :)

Reply:
Let me just copy-paste the message I left in another colleague's PR:

Naming is very important. One of the essential arts of programming.
Splitting code into intelligible chunks (methods), and writing well-structured procedures is yet another)

You might never once in the whole of your career write a new sorting algorithm. You might never create a computer game, a blockchain, or an extra-effective archiver.

But you will be giving names to entities and structuring methods day after day -- until the day of your retirement. 
And possibly this -- and not much more -- will build up to your name as a professional. This will bring you the gratitude of colleagues, or else bring contempt.